README.md

```diff
@@ -175,7 +175,7 @@ Apps people use when taking transit.
 Software that makes it easy to consume GTFS data in a variety of languages.
 
 - [Mapzen GTFS](https://github.com/transitland/mapzen-gtfs) - A Python GTFS library that supports reading individual GTFS tables, or constructing a graph to represent each agency in a feed.
-- [gtfsdb](https://github.com/OpenTransitTools/gtfsdb) - Python library for converting GTFS files into a relational database.
```

Comment:
Instead of removing gtfsdb, I'd prefer to add a note with the caveat you mention in your PR comment (unmaintained, etc.) and link to gtfspy, and we can put it below gtfspy.  Otherwise later someone will ask why it's not on the list.  Does that work for you?